test/test_sparse_csr.py

```diff
@@ -3397,6 +3398,51 @@ def test_triton_bsr_dense_bmm(self, device, dtype, index_dtype, block_size):
                 )
                 self.assertEqual(res_tri, res_dense)
 
+    @onlyCUDA
+    @skipIfRocm
+    @dtypes(torch.half)
+    @dtypesIfCUDA(torch.half)
```

Comment:
One of the two is redundant, isn't it?
```suggestion
    @dtypes(torch.half)
```

Reply:
It throws `AssertionError: @dtypes is mandatory when using @dtypesIf however 'test_triton_bsr_dense_bmm_error_messages' didn't specify it`